src/config/configuration.ts

```diff
@@ -0,0 +1,6 @@
+import { IConfig } from './IConfig';
```

Comment:
IConfig is not used to validate the keys

Reply:
resolved